src/log.py

```diff
@@ -18,16 +19,13 @@ def rotateLogs(self):
 			with gzip.open("logs/wrapper/oldWrapper.log.gz", "w") as f:
 				f.write(originalLog)
 			os.remove("wrapper.log")
-		if self.file:
-			self.file.close()
-			time.sleep(1) # Possibly fix Windows 'in-use' error. Ugh.
 		if os.path.exists("logs/wrapper/current.log"):
 			with open("logs/wrapper/current.log", "r") as f:
 				logData = f.read()
-			with gzip.open("logs/wrapper/%s.log.gz" % time.strftime("%Y-%m-%d_%H-%M-%S"), "w") as f:
-				f.write(logData)
-			os.remove("logs/wrapper/current.log")
-		self.file = open("logs/wrapper/current.log", "a")
+		os.remove("logs/wrapper/current.log")
```

Comment:
Also, the sleep, whether (1) or (60000), does nothing (I tried!).  It is not a timing issue. Something about the way you wrote this maintains a file lock, even if you think it closes.  I don't quite understand what the self.file definition does for you, but (on windows) it seems to keep the file locked.  This code works, although I can understand the desirability of adding some error checking to ensure it doesn't attempt to delete a non existent file...